test/node/node_test.cpp

```diff
@@ -155,19 +163,21 @@ TEST(NodeTest, SimpleSubkeys) {
 }
 
 TEST(NodeTest, StdArray) {
-  std::array<int, 5> evens;
-  evens[0] = 2;
-  evens[1] = 4;
-  evens[2] = 6;
-  evens[3] = 8;
-  evens[4] = 10;
-
+  std::array<int, 5> evens {{ 2, 4, 6, 8, 10 }};
```

Comment:
std::array<int, 5> evens = { 2, 4, 6, 8, 10 };

Actually produces:

    warning: suggest braces around initialization of subobject [-Wmissing-braces]
      std::array<int, 5> evens = { 2, 4, 6, 8, 10 };
                                   ^~~~~~~~~~~~~~
                                   {             }